ruby/ruby/Ice.rb

```diff
@@ -237,7 +237,7 @@ def FormatType.each(&block)
 require_relative 'IceLocal/Communicator.rb'
 require_relative 'IceLocal/EndpointSelectionType.rb'
 require 'Ice/EndpointTypes.rb'
-require_relative 'IceLocal/LocalException.rb'
+require_relative 'IceLocal/LocalExceptions.rb'
```

Comment:
We need to decide where we put these hand-written files. An option is `Ice/`, alongside the generated code.

Reply:
+1 for `Ice/` along generated code.

Reply:
For a follow-up PR.